src/game/g_cmds.c

```diff
@@ -695,8 +696,16 @@ void Cmd_Give_f(gentity_t *ent, unsigned int dwCommand, int value)
 	trap_Argv(++i, amt, sizeof(amt));
 	if (*amt != '\0')
 	{
-		hasAmount = qtrue;
-		amount    = Q_atoi(amt);
+		if (Q_stricmpn(amt, "all", 3) == 0)
+		{
+			hasAmount   = qtrue;
```

Comment:
This allow using "all" as value for skill and health amount. Which is not wanted.

Reply:
Not an issue in new PR